inscriptions/ISic001392.xml

```diff
@@ -191,33 +196,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140899">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TNS">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Ἡ
-                    <supplied reason="omitted">ρό</supplied>
-                    <unclear>δ</unclear>ω<supplied reason="omitted">ρ</supplied>ος
-                    
-                    <lb n="2"/>χρησ<unclear>τ</unclear>ὸς ἔζησε
-                    <lb n="3"/>
-                    <supplied reason="lost">ἔτ</supplied>η Γ<supplied reason="lost">— —</supplied>ΓΙΛΚΕ
-                    <unclear>Φ</unclear>
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — — — —</supplied>ΛΙ#⁷Α
-                    <lb n="5"/>
+                    <lb n="1"/><name>Ἡ<choice cert="low"><corr>ρόδορ</corr><sic>ψωνωφ</sic></choice>ος</name>
+                    <lb n="2"/>χρησ<choice><corr>τ</corr><sic>ι</sic></choice>τὸς ἔ<supplied reason="omitted">ζ</supplied>ησε
+                    <lb n="3"/><supplied reason="lost">ἔτ</supplied>η <num value="3">Γ</num> <space quantity="1" unit="character"/> <g ref="circle"/> <space quantity="1" unit="character"/> <orig>ΓΙΛΚΕ<unclear>Φ</unclear></orig>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><orig>ΛΙ<unclear>Υ</unclear>Α</orig>
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp><app><note>IG 14</note></app></listApp>
```

Comment:
So, the way to handle a case like this is:
a) present in the text division the text interpreted by Kaibel in IG, over lines 1-3, with the later part of line 3 and line 4 presented simply as orig according to Gualtherus ed 2.
b) report in the apparatus, line by line the original antiquarian readings, eg.`<app loc="line 1"><note>Gualtherus 1st ed: ΗΝΩΛΩΙΦΟΣ; Gualtherus 2nd ed: ΗΨΩΝΩΨΟΣ</note></app>` etc.
We do it this way, since we're inferring not that the stone-cutter made a mistake, but that Gualtherus struggled to read what was originally on the stone.